domain/src/main/java/edu/nextstep/camp/calculator/domain/Calculator.kt

```diff
@@ -0,0 +1,24 @@
+package edu.nextstep.camp.calculator.domain
+
+class Calculator {
+    fun calculate(input: String): Int {
+        var accumulated = 0
+        var operationMark = ""
+
+        Generator.getCaculatorInputToTextArr(input).forEach { text ->
+            if (CalculatorInputUtil.isNumberRegex(text)) {
+                accumulated = Operator
+                    .getOperated(
+                        accumulated,
+                        text.toInt(),
+                        operationMark
+                    )
```

Comment:
```suggestion
                    .getOperated(
                        accumulated = accumulated,
                        nextInt = text.toInt(),
                        operatorMark = operationMark
                    )
```
함수의 파라미터만 봐서는 어떤 것을 의미하는지 잘 알기가 어렵습니다.
이럴 경우에는 코틀린의 argument name을 활용하면 가독성이 좋아집니다.